src/openrct2/paint/Paint.Entity.cpp

```diff
@@ -85,11 +85,11 @@ void EntityPaintSetup(paint_session& session, const CoordsXY& pos)
             {
                 continue;
             }
-            if (entityPos.x < gClipSelectionA.x || entityPos.x > gClipSelectionB.x)
+            if (entityPos.x < gClipSelectionA.x || entityPos.x > gClipSelectionB.x + 31)
```

Comment:
No naked numbers, please. As I said, please use `COORDS_XY_STEP - 1`.

Reply:
I think it would be cleaner to use `>=` here and leave out the `- 1`. The same applies to the line below.

Reply:
My bad I misunderstood what you were trying to say with the numbers. I have applied both changes.

Reply:
I disagree, `COORDS_XY_STEP - 1` makes it much clearer it’s confined to this tile.